helm/values.yaml

```diff
@@ -0,0 +1,35 @@
+ingress:
+  enabled: true
+  tls:
+    enabled: true
+  hosts:
+  - localhost
+  path: /
+
+loadBalancer:
+  enabled: true
+
+storage:
+  type: mysql
+  pvcSize: 100Mi
+  migrateVersion: v4.6.2
```

Comment:
Could you also allow a `storageClass` option here?

Reply:
Sure, although the "default" value here would be somewhat tricky because we can set a default for the chart, but the chart doesn't know what the default `StorageClass` is on the target cluster. I guess the way to go here is to **not** set a chart default, and only conditionally set `storageClassName` [here](https://github.com/theupdateframework/notary/pull/1503/files#diff-a6372b10b9b4f207ef15397bd89d20b7R4) if `storage.className` is set?

Reply:
I think maybe that could be something further down the road, chart looks good to me (and seems to work)! I just had a problem with `pod has unbound immediate PersistentVolumeClaims` on my cloud provider which requires me to set the storage class to something different (this happens with lots of Helm charts so it's not unique), particularly I have to set my `volumeBindingMode` so something custom within the storage class.